### PR TITLE
Fix scrolling with mouse wheel in ElementsTowerMenu and ItemStashMenu

### PR DESCRIPTION
**Pull request by Praytic**
Resolves (#332)
giphy

### CODE REVIEW

**Comment by Kvel2D** [comment]
Note to self:
The changes to empty unit buttons are to fix this visual issue:
Bad:
2023-12-19 18_15_17-Window
Good:
2023-12-19 18_16_48-Window

**Comment by Praytic** [comment]
> Note to self: The changes to empty unit buttons are to fix this visual issue: Bad: 2023-12-19 18_15_17-Window Good: 2023-12-19 18_16_48-Window

Fixed "Bad" state of `ItemStashMenu`.

**Review by Kvel2D — APPROVED** [review]

File: Scenes/HUD/ElementTowersMenu.gd [hunk 1]
```diff
@@ -194,6 +192,17 @@ func _update_info_label():
 	_element_info_label.text = text
 
 
+func _update_empty_slots():
```
Comment (Kvel2D) [inline Scenes/HUD/ElementTowersMenu.gd:195]: This function is duplicated here and in item stash. Make it into a static function and call it from both places:

`static func update_empty_slots_generic(empty_slots: Array, element_list: Array)`

**Review by Kvel2D — CHANGES_REQUESTED** [review]

File: Scenes/HUD/ElementTowersMenu.gd (hunk 1 above)
Comment (Praytic) [reply Scenes/HUD/ElementTowersMenu.gd:194]: I have joined them into UnitButtonsContainer script.

**Review by Kvel2D — APPROVED** [review]

**Pull request merged**